rules/aip0192/aip0192.go

```diff
@@ -48,40 +46,3 @@ func isDeprecated(d desc.Descriptor) bool {
 		return false
 	}
 }
-
-func separateInternalComments(comments ...string) struct {
```

Comment:
I think the file that contains this function now in utils/ wasn't added? But otherwise LGTM.